src/Type/Php/ArraySliceFunctionReturnTypeExtension.php

```diff
@@ -62,7 +62,7 @@ public function getTypeFromFunctionCall(
 			$limit = new NullType();
 		}
 
-		$constantArrays = TypeUtils::getConstantArrays($valueType);
```

Comment:
could you give a small explanation what the semantic difference of these 2 get*Array methods is?
If seen you used it in some places while fixing stuff in past PRs, but by reading the code itself I cannot make sense of it.